css-flexbox/align-items.html

```diff
@@ -11,12 +11,15 @@
     /* your styles here 👇 */
     #card-deck {
       display: flex;
-      flex-basis: calc(100% / 3);
-      min-height: 100vh;
       justify-content: center;
       align-items: center;
     }
 
+    .card-container {
+      flex-basis: calc(100% / 3);
+      min-height: 100vh;
```

Comment:
This line should be on `#card-deck`. Did your output look correct after making this change? 🤔

Reply:
I didn't do a double take once I made the changes thought it might magically work for some reason.